getting_started/examples/eval_gr00t_lekiwi.py

```diff
@@ -0,0 +1,508 @@
+
+
+class Gr00tRobotInferenceClient:
+    """Wraps ExternalRobotInferenceClient for LeKiwi observations."""
+
+    def __init__(
+        self,
+        host="localhost",
+        port=5555,
+        language_instruction="Perform the task.",
+        img_size=(480, 640),
+    ):
+        self.language_instruction = language_instruction
+        self.img_size = img_size
+        try:
+            self.policy = ExternalRobotInferenceClient(host=host, port=port)
+            print(f"GR00T Inference Client trying to connect to {host}:{port}")
+            if not self.policy.ping():
+                print(f"W: Could not ping inference server at {host}:{port}.")
+            else:
+                print("Inference server ping successful.")
+        except Exception as e:
+            print(f"Error creating/pinging inference client: {e}")
+            raise
+
+    def get_action_chunk(self, images: dict[str, np.ndarray], state: np.ndarray) -> dict:
+        """Formats observations and calls the GR00T server."""
+        if state.shape != (9,):
+            raise ValueError(f"STATE ERROR: Expected state shape (9,), got {state.shape}")
+
+        default_img = np.zeros(self.img_size + (3,), dtype=np.uint8)
+        # Observation keys sent to the server still use the grouped state keys
+        # as defined in LekiwiDataConfig (assuming that wasn't changed for input)
+        obs_dict = {
+            "video.front": images.get("front", default_img)[np.newaxis, ...],
+            "video.wrist": images.get("wrist", default_img)[np.newaxis, ...],
+            "state.single_arm": state[0:5][np.newaxis, :].astype(np.float64),
+            "state.gripper": state[5:6][np.newaxis, :].astype(np.float64),
+            "state.mobile_base": state[6:9][np.newaxis, :].astype(np.float64),
+            "annotation.human.task_description": [self.language_instruction],
+        }
+
+        try:
+            res = self.policy.get_action(obs_dict)
+            if not isinstance(res, dict) or not res:
+                raise RuntimeError("Invalid/empty response from inference server")
+            # Check if the *returned* action keys match the new expected structure
+            # This depends on the inference server using a LekiwiDataConfig with updated action_keys
+            expected_action_keys = {"action.single_arm", "action.gripper", "action.mobile_base"}
+            if not expected_action_keys.issubset(res.keys()):
+                print(f"Warning: Action chunk missing expected keys. Got: {list(res.keys())}")
+                # Might need error handling depending on strictness
+            #  raise RuntimeError(f"Missing expected action keys in response. Got {res.keys()}")
+            return res
+        except Exception as e:
+            print(f"Error during inference request: {e}")
+            raise
+
+
+#################################################################################
+
+
+_figs = {}
+
+
+def view_img_mpl(img, window_name="MPL Robot View"):
```

Comment:
not a fan of global, can we just use:
```

def view_img(img, img2=None):
    """
    This is a matplotlib viewer since cv2.imshow can be flaky in lerobot env
    also able to overlay the image to ensure camera view is alligned to training settings
    """
    plt.imshow(img)
    if img2 is not None:
        plt.imshow(img2, alpha=0.5)
    plt.axis("off")
    plt.pause(0.001)  # Non-blocking show
    plt.clf()  # Clear the figure for the next frame
```